### DISCUSSION TITLE
Uploading a video with Serverpod

### DISCUSSION THREAD

**Discussion by andriyndev** [serverpod/serverpod#839] [Q&A]
The question is not completely related to Serverpod, rather to Dart libraries. Does someone know a way to upload a video on server with Serverpod with checking video resolution, format, duration on the server side? I checked video processing libraries but it seems like they all don't support Linux/Windows. I'm interested only in libraries with verified publishers otherwise it'd be too risky as for server side.

**Comment by vlidholt** [comment]
There seem to be a few options, like mplayer or ffprobe. You can install them on your machine, then use the dart `Process` class to run them.
